ipc_radio: Correct Kcofing CONFIG_IPC_RADIO_BT_SER choice (#23224)
nrfconnect/sdk-nrf: applications/ipc_radio/README.rst

* :kconfig:option:`CONFIG_IPC_RADIO_802154` - For the IEEE 802.15.4 serialization.

> You can select the Bluetooth Low Energy serialization using the ``CONFIG_IPC_RADIO_BT_SER`` Kconfig choice:

Reviewer: ```suggestion
You can select the Bluetooth Low Energy serialization by setting one of the following Kconfigs:
```

* :kconfig:option:`CONFIG_IPC_RADIO_BT_HCI_IPC` - For the Bluetooth HCI serialization.

> * :kconfig:option:`CONFIG_IPC_RADIO_BT_RPC` - For the Bluetooth host API serialization.

Reviewer 2: Aren't these ``SB_``? https://docs.nordicsemi.com/bundle/ncs-latest/page/kconfig/index.html#!IPC_RADIO_BT_
